src/types/tokens.ts

```diff
@@ -0,0 +1,7 @@
+import { Resource } from 'types/resource';
+
+export interface Tokens extends Resource {
```

Comment:
We can use the singular form for this type, as the access token and refresh token are in the same token record.
```suggestion
export interface Token extends Resource {
```